example/README.md

```diff
@@ -42,28 +31,48 @@ gcloud services enable container.googleapis.com
 gcloud services enable iam.googleapis.com
 ```
 
+## Setup
```

Comment:
I wonder if it's necessary to explain the configuration of remote state - beyond suggesting that people do it?

Reply:
I initially read this as suggesting I should explain remote state more, and added some links. On rereading I'm not so sure haha!

Currently the example `README.md` is aimed at people who are quite new to GKE and Terraform as it runs though setting up `terraform.tfvars`, creating a service account, and creating a remote state GCS bucket. I have tried to keep it concise though and block commands together so you can just copy and paste them if you already know what's going on. I assume people who are familiar with Terraform and GKE will just look at the main `README.md`.

Reply:
Ahh sorry - I understand the confusion. My suggestion was the opposite - that we just assume local state or they manage their state in the way that suits them.

This keeps this page about the module.

Sorry for the confusion.